plugins/package-managers/spdx/src/funTest/kotlin/SpdxDocumentFileFunTest.kt

```diff
@@ -207,6 +207,19 @@ class SpdxDocumentFileFunTest : WordSpec({
                 result.toYaml() should matchExpectedResult(expectedResultFile, definitionFile)
             }
         }
+
+        "handle cycles in dependencies gracefully" {
+            val definitionFile = projectDir.resolve("cyclic-references/project-cyclic.spdx.yml")
+            val expectedResultFile = getAssetFile(
+                "projects/synthetic/spdx-project-cyclic-expected-output.yml"
+            )
+
+            val ortResult = analyze(definitionFile.parentFile, allowDynamicVersions = true)
```

Comment:
Is it necessary to use `analyze()` or could it use `resolveSingleProject()` for a simpler expected result file?